docs/source/building-supergraphs/router.mdx

```diff
@@ -37,32 +37,38 @@ This section walks through setting up a basic graph router using Apollo Server a
 Create a new Node.js project with `npm init`, then install the necessary packages:
 
 ```shell
-npm install @apollo/gateway apollo-server graphql
+npm install @apollo/gateway @apollo/server graphql
 ```
 
 The `@apollo/gateway` package includes the [`ApolloGateway` class](../api/apollo-gateway/). To configure Apollo Server to act as a graph router, you pass an instance of `ApolloGateway` to the `ApolloServer` constructor, like so:
 
-```js {7-11,13-16} title="index.js"
-const { ApolloServer, gql } = require('apollo-server');
+```js title="index.js"
+const { ApolloServer } = require('@apollo/server');
+const { startStandaloneServer } = require('@apollo/server/standalone');
 const { ApolloGateway } = require('@apollo/gateway');
 const { readFileSync } = require('fs');
 
-const supergraphSdl = readFileSync('./supergraph.graphql').toString();
+async function startApolloGateway() {
```

Comment:
If we expect people to maybe copy this code we should call `startApolloGateway` at the end of the snippet.

Reply:
For what it's worth, I'm not sure if what I'm about to say applies to *this* part of the docs, but I know in parts of the docs I worked I explicitly introduced an un-called `async function startServer()` instead of putting the relevant code at the top level in order to *avoid* writing top-level await, since how you call an async function at the top level depends a lot on your build system etc.

Now to be clear, we've had feedback from end users that this is confusing too, so I'm not saying we should stick to it! Although if we're going to use top-level awaits at all, we might not even want to bother with a function at all?

Reply:
I was following your previous example, David! For clarity's sake, I might do away with with `async` function altogether and just use the top-level `await` with a comment noting it (I removed the line highlights below but they are still in the snippet):

```js
import { ApolloServer } from '@apollo/server';
import { startStandaloneServer } from '@apollo/server/standalone';
import { ApolloGateway } from '@apollo/gateway';
import { readFileSync } from 'fs';

const supergraphSdl = readFileSync('./supergraph.graphql').toString();

// Initialize an ApolloGateway instance and pass it
// the supergraph schema as a string
const gateway = new ApolloGateway({
  supergraphSdl,
});

// Pass the ApolloGateway to the ApolloServer constructor
const server = new ApolloServer({
  gateway,
});

// Note the top-level await here!
const { url } = await startStandaloneServer(server);
console.log(`🚀  Server ready at ${url}`);
```

Additionally, I'll update the imports to use ESM syntax so folks can more easily use module syntax. Does that feel like the clearest way to go here @glasser and @trevor-scheer?

Reply:
(Whatever we decide I'll go around and do in every corresponding example)

Reply:
I'm fully onboard (as you might have deduced from similarly timed comments on a different PR)

Reply:
Sounds good.

Reply:
Cool, thank you everyone for weighing in on this one!